app/src/main/java/protect/card_locker/AboutActivity.kt

```diff
@@ -0,0 +1,102 @@
+            .also { this.binding = it }
+        val content = AboutContent(this)
+            .also { this.content = it }
+
+        title = content.pageTitle
+
+        setContentView(binding.root)
+        setSupportActionBar(binding.toolbar)
+        enableToolbarBackButton()
+
+        val copyright = binding.creditsSub
+        copyright.text = content.copyright
+        val versionHistory = binding.versionHistorySub
+        versionHistory.text = content.versionHistory
+
+        binding.versionHistory.tag = "https://catima.app/changelog/"
+        binding.translate.tag = "https://hosted.weblate.org/engage/catima/"
+        binding.license.tag = "https://github.com/CatimaLoyalty/Android/blob/master/LICENSE"
+        binding.repo.tag = "https://github.com/CatimaLoyalty/Android/"
+        binding.privacy.tag = "https://catima.app/privacy-policy/"
+        binding.reportError.tag = "https://github.com/CatimaLoyalty/Android/issues"
+        binding.rate.tag = "https://play.google.com/store/apps/details?id=me.hackerchick.catima"
+
+        bindClickListeners()
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        val id = item.itemId
+        if (id == android.R.id.home) {
+            finish()
+        }
+        return super.onOptionsItemSelected(item)
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        content?.destroy()
+        content = null
+        clearClickListeners()
+        binding = null
+    }
+
+    private fun bindClickListeners() {
+        val binding = binding ?: return
+
+        val openExternalBrowser = View.OnClickListener { view: View ->
+            val tag = view.tag
+            if (tag is String && tag.startsWith("https://")) {
+                OpenWebLinkHandler().openBrowser(this, tag)
+            }
+        }
+
+        binding.versionHistory.setOnClickListener(openExternalBrowser)
+        binding.translate.setOnClickListener(openExternalBrowser)
+        binding.license.setOnClickListener(openExternalBrowser)
+        binding.repo.setOnClickListener(openExternalBrowser)
+        binding.privacy.setOnClickListener(openExternalBrowser)
+        binding.reportError.setOnClickListener(openExternalBrowser)
+        binding.rate.setOnClickListener(openExternalBrowser)
+        binding.credits.setOnClickListener { showCredits() }
+    }
+
+    private fun clearClickListeners() {
+        val binding = binding ?: return
+        binding.versionHistory.setOnClickListener(null)
+        binding.translate.setOnClickListener(null)
+        binding.license.setOnClickListener(null)
+        binding.repo.setOnClickListener(null)
+        binding.privacy.setOnClickListener(null)
+        binding.reportError.setOnClickListener(null)
+        binding.rate.setOnClickListener(null)
+        binding.credits.setOnClickListener(null)
+    }
+
+    private fun showCredits() {
+        val content = content ?: return
```

Comment:
I'm not a fan of this. If content is somehow null, we're just silently not responding to the click on the Credits option. This means the app doesn't do what it should do and also won't crash, meaning I won't be able to see the issue in the Google Play Console crashes meaning the app is misbehaving for users without me ever knowing (as almost nobody ever reports a bug they run into).